inference-engine/src/vpu/common/src/ngraph/transformations/merge_subsequent_dsr_operations.cpp

```diff
@@ -20,13 +20,10 @@ MergeSubsequentDSROperations::MergeSubsequentDSROperations() {
         if (!predecessor) {
             return false;
         }
-        // this will create a new DSR with correct inputs
-        auto newDsr = dsr->copy_with_new_inputs({predecessor->input_value(0), dsr->input_value(1)});
-        // replace DSR2 with new so DSR2 will lose all consumers so it will die after pass execution
-        replace_node(dsr, newDsr);
-        // reconnect all DSR1 consumers even with DSR2 which will be destructed so this is no more an issue
```

Comment:
We already discussed this replacement several times. And each time when you tried to reconnect consumers and inputs `inplace` you got wrong behavior. So the replacement that you remove is working as expected and doesn't require any additional checks.
For example in new replacement you forget to check that predecessor consumer is not old dsr.
So I insist to add set_friendly_name (which is missing here by mistake) instead of changing replacement logic with wrong one.

Reply:
Got it, fixed.